master.xml

```diff
@@ -144,4 +144,8 @@ Your sources have been sync'd successfully.
            name="coreos/shim"
            groups="minilayout" />
 
+  <project path="src/third_party/spec"
```

Comment:
When doing so may need to specify the WORKON_LOCALDIR=appc-spec variable or whatever it is.